### PR TITLE
WIP: Establish visual settings

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Attempt to establish the `display` block and get a feel for how & where it should/could fit in.
For now only setting the `display.name`.

WIP pending discussions

### CODE REVIEW

**Comment by perolavsvendsen** [comment]
#62

File: src/fmu/dataio/_export_item.py [hunk 1]
```diff
@@ -530,8 +531,59 @@ def _fmu_inject_workflow(self):
         """Inject workflow into fmu metadata block."""
         self.dataio._meta_fmu["workflow"] = self.dataio._workflow
 
+    def _display(self):
+        """Process common subfields in the display block.
+        For now, this is simply injecting a skeleton with loose
```
Comment (jcrivenaes) [inline src/fmu/dataio/_export_item.py:536]: Add empty line after docstring punch line (google style)

**Review by jcrivenaes — COMMENTED** [review]
Perhaps we need some (or a bunch of) examples on how display metadata could be? E.g. make examples in the examples folder?

File: src/fmu/dataio/_export_item.py (hunk 1 above)
Comment (perolavsvendsen) [reply src/fmu/dataio/_export_item.py:537]: Added empty line after docstring punch line

**Review by jcrivenaes — APPROVED** [review]
We can give it a go

**Pull request merged**